README.md

```diff
@@ -58,3 +58,5 @@ npm test
 
 
 #### UML
+
+!(REACT_UML)[./REACT_UML.pdf]
```

Comment:
This isn't showing your image because it isn't written correctly, it should be `![React UML](./REACT_UML.pdf)`